src/transformers/testing_utils.py

```diff
@@ -774,7 +774,9 @@ def require_torch_gpu(test_case):
 
 def require_torch_accelerator(test_case):
     """Decorator marking a test that requires an accessible accelerator and PyTorch."""
-    return unittest.skipUnless(torch_device != "cpu", "test requires accelerator")(test_case)
+    return unittest.skipUnless(torch_device is not None and torch_device != "cpu", "test requires accelerator")(
```

Comment:
We need this to avoid incorrect execution of PyTorch related testing suites in TensorFlow environment